JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -146,12 +146,12 @@ InVehicle,Class,ActivityType,IS-A,,,,The device is in a vehicle.,http://www.thew
 OnBicycle,Class,ActivityType,IS-A,,,,The device is on a bicycle.,http://www.theworldavatar.com/kg/ontodevice/,On bicycle
 Running,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is running.,http://www.theworldavatar.com/kg/ontodevice/,Running
 Walking,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is walking.,http://www.theworldavatar.com/kg/ontodevice/,Walking
-Confidence level,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
+ConfidenceLevel,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence level
 hasActivityType,Object Property,,Activity,ActivityType,,only,Links an activity to its corresponding type (e.g., walking, running).,https://www.theworldavatar.com/kg/ontodevice/,has activity type
 hasConfidenceLevel,Object Property,,Activity,ConfidenceLevel,,only,Links an activity to its confidence level.,https://www.theworldavatar.com/kg/ontodevice/,has confidence level
 detectedByDevice, Object Property,,prov:Activity, Device,,only,Relates an activity with the device that detected it.,https://www.theworldavatar.com/kg/ontodevice/,detected by device
-hasValue,Data Property,,ConfidenceLevel,Integer,,only,Represents the numerical confidence value.,https://www.theworldavatar.com/kg/ontodevice/,has value
-hasValue,Data Property,,ActivityType,String,,only,Represents the type of activity as a string.,https://www.theworldavatar.com/kg/ontodevice/,has value
+hasValue,Data Property,,,ConfidenceLevel,Integer,,Represents the numerical confidence value.,https://www.theworldavatar.com/kg/ontodevice/,has value
+hasValue,Data Property,,,ActivityType,String,,Represents the type of activity as a string.,https://www.theworldavatar.com/kg/ontodevice/,has value
```

Comment:
ActivityType hasValue String, is this string used to represent the name of the activity type and its sub classes (e.g. on foot, in vehicle, on bicycle)?

If so, there is a rdfs:label property that can be utilized instead. It is a universal property that is applicable to all classes and this property do not need to be specified in ontodevice.